tests/test_generator.py

```diff
@@ -3,34 +3,37 @@
 import unittest
 from lsst.dax.data_generator import DataGenerator
 import lsst.dax.data_generator.columns as columns
-from lsst.dax.data_generator import Chunker
+from lsst.dax.data_generator import Chunker, UniformSpatialModel
 
 num_stripes = 200
 num_substripes = 5
 chunker = Chunker(0, num_stripes, num_substripes)
 
+# From lsst.sphgeom
+RAD_PER_DEG = 0.0174532925199432957692369076849
 
 class TestDataGenerator(unittest.TestCase):
 
     def testObjectTable(self):
         generator_spec = {
             "Object": {
                 "columns": {"objectId": columns.ObjIdGenerator(),
-                            "ra,decl": columns.RaDecGenerator(chunker),
+                            "ra,decl": columns.RaDecGenerator(),
                             "mag_u,mag_g,mag_r": columns.MagnitudeGenerator(n_mags=3)
-                            }
+                            },
+                "density": UniformSpatialModel(50),
             }
         }
-        chunk_id = 5000
+        chunk_id = 3525
+        seed = 1
 
-        generator = DataGenerator(generator_spec)
+        generator = DataGenerator(generator_spec, chunker, seed=seed)
 
-        seed = 1
         edge_width = 0.017 # degrees
-        data = generator.make_chunk(chunk_id, num_rows=50, seed=seed,
-                                    edge_width=edge_width, edge_only=False)
+        data = generator.make_chunk(chunk_id, edge_width=edge_width, edge_only=False)
+        chunk_area = chunker.getChunkBounds(chunk_id).getArea() / RAD_PER_DEG**2
         self.assertIn('Object', data.keys())
-        self.assertEqual(len(data['Object']), 50)
+        self.assertAlmostEqual(len(data["Object"]), int(50 * chunk_area), delta=1)
```

Comment:
Would a higher density make for a better test by reducing the impact of delta=1?